src/main/java/org/alliancegenome/curation_api/jobs/executors/LoadFileExecutor.java

```diff
@@ -220,7 +221,11 @@ protected <E extends AuditedObject, T extends BaseDTO> boolean runLoad(BaseUpser
 					// e.printStackTrace();
 					history.incrementFailed();
 					addException(history, e.getData());
-				} catch (Exception e) {
+				} catch (KnownIssueValidationException e) { 
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace
<!-- __reviewdog__:ChAxZDE3Y2YwMTAzYTU0MzIzEgpjaGVja3N0eWxl -->